src/content/docs/identityserver/ui/login/dynamicproviders.md

```diff
@@ -9,30 +9,112 @@ redirect_from:
   - /identityserver/v7/ui/login/dynamicproviders/
 ---
 
-Dynamic Identity Providers are a scalable solution for managing authentication with numerous external providers without
-incurring performance penalties or requiring application recompilation. This feature, included in the Enterprise Edition
+Dynamic Identity Providers are a scalable solution for managing authentication with lots of external providers, without
+incurring performance penalties or requiring application recompilation. This feature, included in the [Enterprise Edition](/general/licensing/#enterprise-edition)
 of Duende IdentityServer, enables providers to be configured dynamically from a store at runtime.
 
 ## Dynamic Identity Providers
 
-Normally authentication handlers for external providers are added into your IdentityServer using `AddAuthentication()`
-and `AddOpenIdConnect()`. This is fine for a handful of schemes, but the authentication handler architecture in ASP.NET
-Core was not designed for dozens or more statically registered in the DI system. At some point you will incur a
-performance penalty for having too many. Also, as you need to add or change this configuration you will need to
-re-compile and re-run your startup code for those changes to take effect.
+Authentication handlers for external providers are typically added into your IdentityServer using `AddAuthentication()`
+and `AddOpenIdConnect()`. This is fine for a handful of schemes, but becomes harder to manage if you have too many of them.
```

Comment:
Should we mention that other protocols than OIDC can also be used with dynamic providers. I've done it with Saml2 (there's a sample somewhere)